tox.ini

```diff
@@ -1,4 +1,7 @@
 [tox]
+requires =
+    tox>=4.22 # `dependency_groups` needed
+    tox-uv>=1.19
```

Comment:
Could you help me understand why these requirements are here and not in pyproject?

Reply:
In general case, `tox` version is not managed in pyproject.toml, requirements.txt, etc.
https://github.com/tox-dev/tox-uv/blob/main/.github/workflows/check.yaml#L40
https://github.com/tox-dev/tox-uv/blob/main/tox.ini#L2-L4
I think this is because `tox` is not related to actual implementation of tests.

In our CI, we use `dependency_groups` feature of pyproject.toml and tox, so I added this requirements on tox's configuration.